README-sharing.md

```diff
@@ -0,0 +1,42 @@
+![Javascript ESLint check](https://github.com/CSCfi/swift-x-account-sharing/workflows/Javascript%20ESLint%20check/badge.svg)
+![Python style check](https://github.com/CSCfi/swift-x-account-sharing/workflows/Python%20style%20check/badge.svg)
+![Python Unit Tests](https://github.com/CSCfi/swift-x-account-sharing/workflows/Python%20Unit%20Tests/badge.svg)
+
+### swift-x-account-sharing – OS swift container sharing backend
+Openstack Swift Access Control Lists don't natively implement querying
+for containers that are shared for specific user. (i.e. if the user has
+access to these containers on the object storage platform) This backend
+is meant to be used as a workaround to implement the functionality, as
+it is needed in some use cases.
+
+The backend contains a database for managing the information on which
+containers have been shared, by whom and to whom. The backend is meant
+to be used as a solution to centrally share the container access
+information in a given user pool, and thus isn't baked into the Openstack
+native command line tools.
+
+The default port for the service is 9090.
+
+### Usage
+```pip install . && swift-x-account-sharing```
```

Comment:
is this still valid

Reply:
this is valid, setup gives CLI entry points for all the subpackages with old syntax